speechbrain/utils/train_logger.py

```diff
@@ -178,17 +177,12 @@ def log_figure(self, name, value):
 
 class WandBLogger(TrainLogger):
     """Logger for wandb. To be used the same way as TrainLogger. Handles nested dicts as well.
-    An example on how to use this can be found in recipes/Voicebank/MTL/CoopNet/"""
+    An example on how to use this can be found in recipes/Voicebank/MTL/CoopNet/
```

Comment:
could you please add the docstring for arguments / a toy example to demonstrate how to use it?